src/bin/noir_cli/cli.rs

```diff
@@ -43,36 +43,10 @@ pub enum Commands {
         #[arg(long)]
         output: Option<PathBuf>,
     },
-    /// Take in vk as generated by Noir toolchain and output it in hex format.
-    KeyToHex {
-        /// Input file for verification key
-        #[arg(long)]
-        input: PathBuf,
-
-        /// Output file for verification key in hex format [or stdout if not specified]
-        #[arg(long)]
-        output: Option<PathBuf>,
-    },
-    /// Process proof data from JSON file and generate output files for proof and
-    /// public inputs suitable to be submitted to zkVerify.
-    /// To be used for Noir versions <= 0.30.0.
-    ProofData {
-        /// Input file for proof and public inputs
-        #[arg(long)]
-        input_json: PathBuf,
-
-        /// Output file for proof data [or stdout if not specified]
-        #[arg(long)]
-        output_proof: Option<PathBuf>,
-
-        /// Output file for verification key [or stdout if not specified]
-        #[arg(long)]
-        output_pubs: Option<PathBuf>,
-    },
     /// Parse proof as generated by bb tool from Noir toolchain and output files for proof and
     /// public inputs suitable to be submitted to zkVerify. Both in hex and binary format.
     /// To be used for Noir versions >= 0.31.0.
```

Comment:
is `>= 0.31.0` still true?

Reply:
Yes, but moved the piece of info to the top level description